frontera-cms/templates/home.html

```diff
@@ -0,0 +1,24 @@
+{% extends "./fullwidth.html" %}
+{% load cms_tags staticfiles %}
+
+{% block assets_custom %}
+  <link rel="stylesheet" href="{% static 'frontera-cms/css/build/site.css' %}">
+  <link rel="stylesheet" href="{% static 'frontera-cms/css/build/site.header.css' %}">
```

Comment:
__Why Not Make `fullwidth.html` Use These?__
Because the rest of Frontera's pages are not ready for the redesign changes these bring.